build_image.sh

```diff
@@ -170,6 +183,13 @@ elif [ "$IMAGE_TYPE" = "aboot" ]; then
     zip -g $ABOOT_BOOT_IMAGE version
     rm version
 
+    generate_onie_device_list
+    if [ -f ./installer/x86_64/devices/platforms_asic ]; then
+        cp ./installer/x86_64/devices/platforms_asic .platforms_asic
+        zip -g $OUTPUT_ABOOT_IMAGE .platforms_asic
+        rm .platforms_asic
+    fi
```

Comment:
Implement a new function `generate_aboot_device_list` ? If some logic is common, you can extract into another function like  `generate_device_list`.

Reply:
I think it should be fine. 
The extra code here is just for the case that if aboot has a generic build. We should simply bypass the cp process.